apps/studio/components/interfaces/Home/ServiceStatus.tsx

```diff
@@ -40,23 +43,61 @@ const ServiceStatus = () => {
   const storageStatus = status?.find((service) => service.name === 'storage')
 
   // [Joshen] Need individual troubleshooting docs for each service eventually for users to self serve
-  const services = [
+  const services: {
+    name: string
+    error?: string
+    docsUrl?: string
+    isLoading: boolean
+    isSuccess?: boolean
+  }[] = [
     {
       name: 'Database',
-      docsUrls: undefined,
+      error: undefined,
+      docsUrl: undefined,
       isLoading: isLoadingPostgres,
       isSuccess: isSuccessPostgres,
     },
-    { name: 'PostgREST', docsUrls: undefined, isLoading, isSuccess: restStatus?.healthy },
+    {
+      name: 'PostgREST',
+      error: restStatus?.error,
+      docsUrl: undefined,
+      isLoading,
+      isSuccess: restStatus?.healthy,
+    },
     ...(authEnabled
-      ? [{ name: 'Auth', docsUrls: undefined, isLoading, isSuccess: authStatus?.healthy }]
+      ? [
+          {
+            name: 'Auth',
+            error: authStatus?.error,
+            docsUrl: undefined,
+            isLoading,
+            isSuccess: authStatus?.healthy,
+          },
+        ]
       : []),
-    { name: 'Realtime', docsUrls: undefined, isLoading, isSuccess: realtimeStatus?.healthy },
-    // { name: 'Storage', docsUrls: undefined, isLoading, isSuccess: storageStatus?.healthy },
+    ...(realtimeEnabled
+      ? [
+          {
+            name: 'Realtime',
+            error: realtimeStatus?.error,
+            docsUrl: undefined,
+            isLoading,
+            isSuccess: realtimeStatus?.healthy,
+          },
+        ]
+      : []),
+    {
+      name: 'Storage',
+      error: storageStatus?.error,
+      docsUrl: undefined,
+      isLoading,
+      isSuccess: storageStatus?.healthy,
+    },
```

Comment:
Should this be conditional by wrapping in `storageEnabled`?

Reply:
oh yes good point!